src/decpowof2.c

```diff
@@ -27,9 +27,8 @@
  *  2 are needed as negative exponents are handled via divide. Entries
  *  1023-1076 are required to handle denormal doubles (-1022 - 54)
 */
-#ifdef __clang__
+#ifndef PVECLIB_DISABLE_DFP
 // clang does not support Decimal Floating Point at this time.
```

Comment:
Now that you have it all tested in the 'configure' script, do you want to call out clang here? In other words, you could delete this comment.

Reply:
Why yes, Yes I do!.

But you or you think others will find this offensive. I can remove it. 

Otherwise I intend to leave this comment in this source. Otherwise I would expect another set of questions.